StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1121UnitTests.cs

```diff
@@ -668,6 +668,9 @@ public class Bar {{ }}
         [MemberData(nameof(AllTypes))]
         public async Task TestNameOfAsync(string predefined, string fullName)
         {
+            // Currently unused
```

Comment:
💡 Maybe better to say "Not needed for this test" for these.

Reply:
done 😉